Lib/csv.py

```diff
@@ -398,13 +398,12 @@ def has_header(self, sample):
         # subtracting from the likelihood of the first row being a header.
 
         rdr = reader(StringIO(sample), self.sniff(sample))
-
         header = next(rdr) # assume first row is header
-
         columns = len(header)
         columnTypes = {}
+        average_size = 0 
+        col_are_strings = True
         for i in range(columns): columnTypes[i] = None
-
```

Comment:
I think the existing blank lines were useful to visually separate parts of the method, could you restore them?
(but not with extra spaces, just blank line)